mappings/net/minecraft/client/render/entity/EnderCrystalEntityRenderer.mapping

```diff
@@ -1,3 +1,10 @@
 CLASS net/minecraft/class_892 net/minecraft/client/render/entity/EnderCrystalEntityRenderer
+	FIELD field_21002 SIN_PI_OVER_FOUR F
```

Comment:
why not `INVERSED_SQRT_OF_TWO`

Reply:
The constant is initialized as `sin(pi/4)`.